src/main/webapp/app/shared/delete-dialog/delete-dialog.component.html

```diff
@@ -29,6 +29,23 @@ <h4 class="modal-title">
     </div>
     <div class="modal-body">
         <p [jhiTranslate]="deleteQuestion" [translateValues]="translateValues">Are you sure you want to delete?</p>
+
+        @if (entitySummaryTitle) {
+            <b [jhiTranslate]="entitySummaryTitle">Summary</b>
+        }
+        @if (entitySummary) {
+            <ul>
+                @for (summaryKey of objectKeys(entitySummary); track summaryKey; let i = $index) {
+                    @if (entitySummary[summaryKey] !== null) {
```

Comment:
Do we actually get `null` values here or is it `undefined` that we should be looking out for? I ask because we typically don't use `null`